manifests/crane.pp

```diff
@@ -20,17 +22,23 @@
 #
 # $ssl_protocol::               SSLProtocol configuration to use
 class pulp::crane (
-  Boolean $debug = $::pulp::crane::params::debug,
-  Integer[0, 65535] $port = $::pulp::crane::params::port,
-  Stdlib::Absolutepath $data_dir = $::pulp::crane::params::data_dir,
-  Integer[0] $data_dir_polling_interval = $::pulp::crane::params::data_dir_polling_interval,
-  Stdlib::Absolutepath $key = $::pulp::crane::params::key,
-  Stdlib::Absolutepath $cert = $::pulp::crane::params::cert,
-  Stdlib::Absolutepath $ca_cert = $::pulp::crane::params::ca_cert,
-  Optional[String] $ssl_protocol = $::pulp::crane::params::ssl_protocol,
-) inherits pulp::crane::params {
-  class { '::pulp::crane::install': } ~>
-  class { '::pulp::crane::config': } ~>
-  class { '::pulp::crane::apache': } ->
-  Class['pulp::crane']
+  Stdlib::Absolutepath $key,
+  Stdlib::Absolutepath $cert,
+  Stdlib::Absolutepath $ca_cert,
+
+  Boolean                        $debug                     = false,
```

Comment:
I've stopped caring about alignment because I think it creates too much noise in changes. With Puppet 4 types it can also hinder readability instead of add to it IMHO.

Reply:
Alignment removed.